### PR TITLE
@sweir27 => Be more defensive about partnerless artworks

### PR DESCRIPTION
**Pull request by mzikherman**
Fixes a couple of Sentry issues for MP.

This is on top of https://github.com/artsy/metaphysics/pull/969

### CODE REVIEW

File: src/schema/artwork/meta.js [hunk 1]
```diff
@@ -15,8 +15,10 @@ const forSaleIndication = artwork =>
 
 const dimensions = artwork => artwork.dimensions[artwork.metric]
 
-const partnerDescription = ({ partner: { name }, forsale }) => {
-  if (!name) return undefined
+const partnerDescription = ({ partner, forsale }) => {
+  if (isEmpty(partner)) return undefined
+  const name = partner.name
```
Comment (sweir27) [inline src/schema/artwork/meta.js]: Can you just do `const name = partner && partner.name`?

Comment (mzikherman) [reply src/schema/artwork/meta.js]: I'll try!

Comment (mzikherman) [reply src/schema/artwork/meta.js]: Yup, looks like it!

File: src/schema/artwork/meta.js [hunk 2]
```diff
@@ -15,8 +15,9 @@ const forSaleIndication = artwork =>
 
 const dimensions = artwork => artwork.dimensions[artwork.metric]
 
-const partnerDescription = ({ partner: { name }, forsale }) => {
-  if (!name) return undefined
+const partnerDescription = ({ partner, forsale }) => {
+  const name = partner && partner.name
+  if (isEmpty(name)) return undefined
```
Comment (sweir27) [inline src/schema/artwork/meta.js]: Also! Any reason to return `undefined` here instead of `null`?

Comment (mzikherman) [reply src/schema/artwork/meta.js]: No real reason! Either should work, so I just left it as-is, it's just a helper method used by the resolver.

**Comment by sweir27** [comment]
Cool, thanks for the fix!

**Pull request merged**